src/main/kotlin/racingcar/resources/Constants.kt

```diff
@@ -0,0 +1,35 @@
+package racingcar.resources
+
+object Comments {
+    const val ENTER_CAR_NAME_COMMENT = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)"
+    const val NUMBER_OF_ATTEMPT_COMMENT = "시도할 횟수는 몇 회인가요?"
+    const val GAME_RESULT_COMMENT = "\n실행 결과"
+    const val FINAL_WINNER_COMMENT = "최종 우승자 : "
+    const val SEPARATOR = " : "
+    const val MOVEMENT = "-"
+}
+
+object Number {
+    const val ZERO = 0
+    const val NINE = 9
+}
+
+object GameValue {
+    const val INITIAL_MOVEMENT_VALUE = 0
+    const val MOVEMENT_VALUE = 1
+    const val MOVEMENT_THRESHOLD_VALUE = 4
+}
+
+object ValidationValue {
+    const val MIN_NAME_VALUE = 1
+    const val MAX_NAME_VALUE = 5
+
+    const val MIN_CAR_COUNT_VALUE = 1
+}
+
+object Error {
+    const val ZERO_CAR_ERROR = "경주를 할 자동차가 없습니다."
+    const val NAME_LENGTH_ERROR = "자동차 이름의 길이가 1 ~ 5 사이가 아닙니다."
```

Comment:
1 ~ 5 도 아래와 같이 수정하시면 ValidationValue 만 수정해도 메세지가 변경되어 유지보수시 서로의 값이 달라지는 것을 막을 수 있어 보입니다.
```suggestion
    const val NAME_LENGTH_ERROR = "자동차 이름의 길이가 $ValidationValue.MIN_NAME_VALUE ~ $ValidationValue.MAX_NAME_VALUE  사이가 아닙니다."
```

Reply:
개인적으로 한 파일에서만 쓰이는 값들은 그 파일 안에 가깝게 위치 시켰을 때, 가독성과 유지 보수가 더 용이하다고 생각합니다. 이에 대해 준형님은 어떻게 생각하시나요?

Reply:
저는 그냥 상수들을 한곳에 관리하는게 나중에 처리하기 편해서 그렇게 했습니다. 

안드로이드 개발하면서도 string/xml 파일에서 관리하는게 습관이 돼서 그런것 같습니다.